spf.gemspec

```diff
@@ -38,35 +39,31 @@ Gem::Specification.new do |s|
   ]
   s.homepage = "https://github.com/agaridata/spf-ruby"
   s.licenses = ["none (all rights reserved)"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23.2"
+  s.rubygems_version = "2.4.6"
   s.summary = "Implementation of the Sender Policy Framework"
 
   if s.respond_to? :specification_version then
-    s.specification_version = 3
+    s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
       s.add_runtime_dependency(%q<ruby-ip>, ["~> 0.9.1"])
-      s.add_development_dependency(%q<rspec>, ["~> 2.8.0"])
-      s.add_development_dependency(%q<rdoc>, ["~> 3.12"])
-      s.add_development_dependency(%q<bundler>, ["~> 1.0"])
-      s.add_development_dependency(%q<jeweler>, ["~> 1.8.7"])
-      s.add_development_dependency(%q<ruby-ip>, ["~> 0.9.1"])
+      s.add_development_dependency(%q<rspec>, ["~> 2.9"])
+      s.add_development_dependency(%q<rdoc>, ["~> 3"])
+      s.add_development_dependency(%q<bundler>, ["~> 1.2"])
+      s.add_development_dependency(%q<jeweler>, ["~> 1.8"])
```

Comment:
Sounds good, let's downgrade `bundler` back to 1.0 and keep the other changes.

I'm also wondering whether we can consolidate this and the following `if` branches, since they seem to be largely the same? No need for those case distinctions, huh?